packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -517,6 +560,49 @@ class WebpackCLI implements IWebpackCLI {
     return info;
   }
 
+  async getConfigOutput(options: WebpackDevServerOptions): Promise<void> {
+    const config = await this.loadConfig({
+      config: [options.file ? options.file : "webpack.config.js"],
+      name: options.name,
+      configName: options.name ? (options.name as unknown as string[]) : undefined,
+    });
+    if (options.name) {
+      config.options = Array.isArray(config.options)
+        ? config.options.filter((option: any) => options.name?.includes(option.name))
+        : [];
+    }
+
+    if (options.include) {
+      const includeOptions = webpack.config.getNormalizedWebpackOptions({});
+      Array.isArray(config.options)
+        ? config.options.forEach((option) => {
+            Object.keys(option || {}).forEach((key) => {
+              if (options.include.includes(key) && options.name?.includes(option.name || "")) {
+                let finalOption: any = includeOptions[key as keyof WebpackOptionsNormalized];
+                if (typeof option[key as keyof typeof option] === "string") {
+                  finalOption = option[key as keyof typeof option] as string;
+                } else {
+                  Object.keys(option[key as keyof typeof option] || {}).forEach((subkey) => {
+                    finalOption[subkey] = (option[key as keyof typeof option] as any)?.[subkey];
+                  });
+                }
+                console.log(`\n${key}:`, finalOption);
+              }
+            });
+          })
+        : [];
+    }
+
+    const compiler = webpack(config.options as webpack.Configuration[]);
+    compiler.run((err, stats) => {
```

Comment:
We don't need to run this

Reply:
No it's running only in case of --file and --name for --inckude I will remove it